rust/lance-index/src/scalar/inverted/tokenizer.rs

```diff
@@ -141,9 +145,72 @@ fn build_base_tokenizer_builder(name: &str) -> Result<tantivy::tokenizer::TextAn
             tantivy::tokenizer::RawTokenizer::default(),
         )
         .dynamic()),
+        #[cfg(feature = "tokenizer-lindera")]
+        s if s.starts_with("lindera/") => {
+            return build_lindera_tokenizer_builder(s);
+        }
         _ => Err(Error::invalid_input(
             format!("unknown base tokenizer {}", name),
             location!(),
         )),
     }
 }
+
+lazy_static::lazy_static! {
+    pub static ref LANCE_TOKENIZER_HOME: Option<PathBuf> = LANCE_HOME.as_ref().map(|p| p.join("tokenizers"));
+}
+
+#[cfg(feature = "tokenizer-lindera")]
+fn build_lindera_tokenizer_builder(dic: &str) -> Result<tantivy::tokenizer::TextAnalyzerBuilder> {
```

Comment:
What does `dic` stand for?

Reply:
this dic is used as relative model path. it seems that tokenizer name will be written into final index file, so we cannot use absolute path. so I use relative path to $LANCE_HOME.

Reply:
Could you use a better variable name? `dictionary_path` or `lang_model_path` or something similar? I don't think most people will understand what `dic` means.